contrib/apelscripts/condor_batch_blah.py

```diff
@@ -130,6 +135,13 @@ def condor_q_format(job_history: Path, *formats: str) -> str:
 else:
     batch_stream = blah_stream = sys.stdout
 
+accounting_user_expr = "ifThenElse(!isUndefined(x509userproxysubject), x509userproxysubject, orig_AuthTokenSubject)"
+accounting_group_expr = "ifThenElse(!isUndefined(x509UserProxyFirstFQAN), x509UserProxyFirstFQAN, %s)"
+if use_wlcg_groups:
+    accounting_group_expr = accounting_group_expr % 'ifThenElse(!isUndefined(orig_AuthTokenGroups), split(orig_AuthTokenGroups, ",")[0], %s)'
+accounting_group_expr = accounting_group_expr % 'ifThenElse(!isUndefined(userMap("ApelAGMap", Owner)), userMap("ApelAGMap", Owner), %s)'
```

Comment:
If you want to express "if `A isnt undefined` then `A` else `B`" you can use the [ternary operator shorthand](https://htcondor.readthedocs.io/en/latest/classads/classad-mechanism.html#classad-operators) `A ?: B`. This is already used above in `format_apel_scaling` to build a similar cascade.
```suggestion
accounting_group_expr = accounting_group_expr % '(userMap("ApelAGMap", Owner) ?: %s)'
```